module,execution_time (sec),success,timestamp,error,error_short
cowidev.vax.batch.argentina,4.36,True,2022-10-11T07:00:16,,
cowidev.vax.batch.australia,2.33,True,2022-10-11T07:00:14,,
cowidev.vax.batch.austria,3.28,True,2022-10-11T07:00:15,,
cowidev.vax.batch.belgium,2.83,True,2022-10-11T07:00:14,,
cowidev.vax.batch.bolivia,2.76,True,2022-10-11T07:00:14,,
cowidev.vax.batch.canada,6.03,True,2022-10-11T07:00:20,,
cowidev.vax.batch.chile,0.76,True,2022-10-11T07:00:15,,
cowidev.vax.batch.czechia,3.87,True,2022-10-11T07:00:18,,
cowidev.vax.batch.denmark,4.4,True,2022-10-11T07:00:19,,
cowidev.vax.batch.ecdc,149.68,True,2022-10-11T07:02:45,,
cowidev.vax.batch.ecuador,1.79,True,2022-10-11T07:00:17,,
cowidev.vax.batch.estonia,0.67,True,2022-10-11T07:00:18,,
cowidev.vax.batch.france,0.47,True,2022-10-11T07:00:18,,
cowidev.vax.batch.germany,2.27,True,2022-10-11T07:00:20,,
cowidev.vax.batch.greece,3.01,True,2022-10-11T07:00:22,,
cowidev.vax.batch.hong_kong,13.25,True,2022-10-11T07:00:32,,
cowidev.vax.batch.indonesia,,,2022-10-11T07:00:20,,
cowidev.vax.batch.ireland,0.72,True,2022-10-11T07:00:20,,
cowidev.vax.batch.israel,12.24,True,2022-10-11T07:00:33,,
cowidev.vax.batch.italy,12.95,True,2022-10-11T07:00:33,,
cowidev.vax.batch.japan,1.3,False,2022-10-11T07:04:02,"Traceback (most recent call last):
  File ""/mnt/owid_live_covid/covid-19-data/scripts/src/cowidev/cmd/commons/get.py"", line 46, in run
    module.main()
  File ""/mnt/owid_live_covid/covid-19-data/scripts/src/cowidev/vax/batch/japan.py"", line 236, in main
    Japan().export()
  File ""/mnt/owid_live_covid/covid-19-data/scripts/src/cowidev/vax/batch/japan.py"", line 220, in export
    df = self.read().pipe(self.pipeline_base)
  File ""/mnt/owid_live_covid/covid-19-data/scripts/src/cowidev/vax/batch/japan.py"", line 156, in read
    df_latest = self.read_latest().pipe(self.pipe_latest)
  File ""/mnt/owid_live_covid/covid-19-data/scripts/src/cowidev/vax/batch/japan.py"", line 88, in read_latest
    dfs.append(self._read_xlsx(self.source_url, self.sheets, self.metrics))
  File ""/mnt/owid_live_covid/covid-19-data/scripts/src/cowidev/vax/batch/japan.py"", line 97, in _read_xlsx
    xlsx = pd.ExcelFile(url)
  File ""/mnt/owid_live_covid/covid-19-data/scripts/venv/lib/python3.9/site-packages/pandas/io/excel/_base.py"", line 1191, in __init__
    ext = inspect_excel_format(
  File ""/mnt/owid_live_covid/covid-19-data/scripts/venv/lib/python3.9/site-packages/pandas/io/excel/_base.py"", line 1070, in inspect_excel_format
    with get_handle(
  File ""/mnt/owid_live_covid/covid-19-data/scripts/venv/lib/python3.9/site-packages/pandas/io/common.py"", line 609, in get_handle
    ioargs = _get_filepath_or_buffer(
  File ""/mnt/owid_live_covid/covid-19-data/scripts/venv/lib/python3.9/site-packages/pandas/io/common.py"", line 312, in _get_filepath_or_buffer
    with urlopen(req_info) as req:
  File ""/mnt/owid_live_covid/covid-19-data/scripts/venv/lib/python3.9/site-packages/pandas/io/common.py"", line 212, in urlopen
    return urllib.request.urlopen(*args, **kwargs)
  File ""/home/owid/.pyenv/versions/3.9.6/lib/python3.9/urllib/request.py"", line 214, in urlopen
    return opener.open(url, data, timeout)
  File ""/home/owid/.pyenv/versions/3.9.6/lib/python3.9/urllib/request.py"", line 523, in open
    response = meth(req, response)
  File ""/home/owid/.pyenv/versions/3.9.6/lib/python3.9/urllib/request.py"", line 632, in http_response
    response = self.parent.error(
  File ""/home/owid/.pyenv/versions/3.9.6/lib/python3.9/urllib/request.py"", line 561, in error
    return self._call_chain(*args)
  File ""/home/owid/.pyenv/versions/3.9.6/lib/python3.9/urllib/request.py"", line 494, in _call_chain
    result = func(*args)
  File ""/home/owid/.pyenv/versions/3.9.6/lib/python3.9/urllib/request.py"", line 641, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 404: Not Found
",HTTP Error 404: Not Found
cowidev.vax.batch.jersey,0.93,True,2022-10-11T07:00:33,,
cowidev.vax.batch.latvia,22.72,True,2022-10-11T07:00:55,,
cowidev.vax.batch.lithuania,2.08,True,2022-10-11T07:00:35,,
cowidev.vax.batch.luxembourg,1.82,True,2022-10-11T07:00:35,,
cowidev.vax.batch.malaysia,1.23,True,2022-10-11T07:00:35,,
cowidev.vax.batch.malta,0.75,True,2022-10-11T07:00:36,,
cowidev.vax.batch.netherlands,,,2022-10-11T07:00:35,,
cowidev.vax.batch.new_zealand,17.48,True,2022-10-11T07:00:53,,
cowidev.vax.batch.norway,0.37,True,2022-10-11T07:00:36,,
cowidev.vax.batch.peru,5.33,True,2022-10-11T07:00:41,,
cowidev.vax.batch.portugal,,,2022-10-11T07:00:36,,
cowidev.vax.batch.romania,19.35,True,2022-10-11T07:00:55,,
cowidev.vax.batch.saudi_arabia,12.1,True,2022-10-11T07:00:53,,
cowidev.vax.batch.singapore,2.1,True,2022-10-11T07:00:55,,
cowidev.vax.batch.slovakia,94.96,True,2022-10-11T07:02:28,,
cowidev.vax.batch.slovenia,2.28,True,2022-10-11T07:00:57,,
cowidev.vax.batch.south_korea,82.09,True,2022-10-11T07:02:17,,
cowidev.vax.batch.spc,19.02,True,2022-10-11T07:01:14,,
cowidev.vax.batch.sweden,20.06,True,2022-10-11T07:01:17,,
cowidev.vax.batch.switzerland,75.73,True,2022-10-11T07:02:30,,
cowidev.vax.batch.trinidad_and_tobago,2.7,True,2022-10-11T07:01:20,,
cowidev.vax.batch.ukraine,,,2022-10-11T07:01:20,,
cowidev.vax.batch.united_kingdom,14.93,True,2022-10-11T07:01:35,,
cowidev.vax.batch.united_states,74.57,True,2022-10-11T07:02:49,,
cowidev.vax.batch.uruguay,6.08,True,2022-10-11T07:02:23,,
cowidev.vax.batch.zimbabwe,1.07,True,2022-10-11T07:02:25,,
cowidev.vax.incremental.africacdc,,,2022-10-11T07:02:25,,
cowidev.vax.incremental.antigua_barbuda,,,2022-10-11T07:02:25,,
cowidev.vax.incremental.aruba,2.7,True,2022-10-11T07:02:27,,
cowidev.vax.incremental.azerbaijan,,,2022-10-11T07:02:27,,
cowidev.vax.incremental.bahrain,1.51,True,2022-10-11T07:02:29,,
cowidev.vax.incremental.bangladesh,23.51,True,2022-10-11T07:02:52,,
cowidev.vax.incremental.barbados,,,2022-10-11T07:02:29,,
cowidev.vax.incremental.brazil,0.08,True,2022-10-11T07:02:29,,
cowidev.vax.incremental.bulgaria,5.88,True,2022-10-11T07:02:35,,
cowidev.vax.incremental.china,60.08,True,2022-10-11T07:03:30,,
cowidev.vax.incremental.costa_rica,,,2022-10-11T07:02:35,,
cowidev.vax.incremental.croatia,0.31,True,2022-10-11T07:02:35,,
cowidev.vax.incremental.cuba,3.34,True,2022-10-11T07:02:38,,
cowidev.vax.incremental.curacao,0.44,True,2022-10-11T07:02:39,,
cowidev.vax.incremental.cyprus,1.34,True,2022-10-11T07:02:40,,
cowidev.vax.incremental.dominican_republic,6.45,True,2022-10-11T07:02:47,,
cowidev.vax.incremental.el_salvador,0.69,True,2022-10-11T07:02:45,,
cowidev.vax.incremental.faeroe_islands,0.44,True,2022-10-11T07:02:46,,
cowidev.vax.incremental.fiji,0.0,True,2022-10-11T07:02:46,,
cowidev.vax.incremental.finland,0.27,True,2022-10-11T07:02:46,,
cowidev.vax.incremental.gabon,,,2022-10-11T07:02:46,,
cowidev.vax.incremental.georgia,0.99,True,2022-10-11T07:02:47,,
cowidev.vax.incremental.greenland,0.8,True,2022-10-11T07:02:47,,
cowidev.vax.incremental.guatemala,,,2022-10-11T07:02:47,,
cowidev.vax.incremental.guernsey,0.4,True,2022-10-11T07:02:47,,
cowidev.vax.incremental.hungary,1.05,True,2022-10-11T07:02:48,,
cowidev.vax.incremental.iceland,0.34,True,2022-10-11T07:02:48,,
cowidev.vax.incremental.india,0.89,True,2022-10-11T07:02:49,,
cowidev.vax.incremental.iran,,,2022-10-11T07:02:48,,
cowidev.vax.incremental.isle_of_man,1.01,True,2022-10-11T07:02:49,,
cowidev.vax.incremental.jamaica,1.99,True,2022-10-11T07:02:51,,
cowidev.vax.incremental.kazakhstan,65.03,True,2022-10-11T07:03:54,,
cowidev.vax.incremental.kosovo,1.55,True,2022-10-11T07:02:51,,
cowidev.vax.incremental.kyrgyzstan,0.99,True,2022-10-11T07:02:52,,
cowidev.vax.incremental.laos,2.31,True,2022-10-11T07:02:53,,
cowidev.vax.incremental.macao,20.21,False,2022-10-11T07:04:22,"Traceback (most recent call last):
  File ""/mnt/owid_live_covid/covid-19-data/scripts/src/cowidev/cmd/commons/get.py"", line 46, in run
    module.main()
  File ""/mnt/owid_live_covid/covid-19-data/scripts/src/cowidev/vax/incremental/macao.py"", line 86, in main
    Macao().export()
  File ""/mnt/owid_live_covid/covid-19-data/scripts/src/cowidev/vax/incremental/macao.py"", line 72, in export
    data = self.read()
  File ""/mnt/owid_live_covid/covid-19-data/scripts/src/cowidev/vax/incremental/macao.py"", line 20, in read
    elem = driver.find_element_by_partial_link_text(""Weekly Bulletin on COVID-19"")
  File ""/mnt/owid_live_covid/covid-19-data/scripts/venv/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py"", line 462, in find_element_by_partial_link_text
    return self.find_element(by=By.PARTIAL_LINK_TEXT, value=link_text)
  File ""/mnt/owid_live_covid/covid-19-data/scripts/venv/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py"", line 976, in find_element
    return self.execute(Command.FIND_ELEMENT, {
  File ""/mnt/owid_live_covid/covid-19-data/scripts/venv/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py"", line 321, in execute
    self.error_handler.check_response(response)
  File ""/mnt/owid_live_covid/covid-19-data/scripts/venv/lib/python3.9/site-packages/selenium/webdriver/remote/errorhandler.py"", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {""method"":""partial link text"",""selector"":""Weekly Bulletin on COVID-19""}
  (Session info: headless chrome=105.0.5195.102)

","Message: no such element: Unable to locate element: {""method"":""partial link text"",""selector"":""Weekly Bulletin on COVID-19""}
  (Session info: headless chrome=105.0.5195.102)
"
cowidev.vax.incremental.moldova,,,2022-10-11T07:02:52,,
cowidev.vax.incremental.monaco,30.96,True,2022-10-11T07:03:23,,
cowidev.vax.incremental.mongolia,0.71,True,2022-10-11T07:02:54,,
cowidev.vax.incremental.montenegro,0.1,True,2022-10-11T07:02:54,,
cowidev.vax.incremental.morocco,,,2022-10-11T07:02:54,,
cowidev.vax.incremental.myanmar,,,2022-10-11T07:02:54,,
cowidev.vax.incremental.nepal,,,2022-10-11T07:02:54,,
cowidev.vax.incremental.north_macedonia,,,2022-10-11T07:02:54,,
cowidev.vax.incremental.northern_cyprus,0.86,True,2022-10-11T07:02:55,,
cowidev.vax.incremental.paho,65.56,True,2022-10-11T07:04:00,,
cowidev.vax.incremental.pakistan,1.5,True,2022-10-11T07:03:13,,
cowidev.vax.incremental.philippines,,,2022-10-11T07:03:13,,
cowidev.vax.incremental.poland,3.66,True,2022-10-11T07:03:17,,
cowidev.vax.incremental.qatar,,,2022-10-11T07:03:17,,
cowidev.vax.incremental.russia,0.54,False,2022-10-11T07:04:23,"Traceback (most recent call last):
  File ""/mnt/owid_live_covid/covid-19-data/scripts/venv/lib/python3.9/site-packages/urllib3/connectionpool.py"", line 699, in urlopen
    httplib_response = self._make_request(
  File ""/mnt/owid_live_covid/covid-19-data/scripts/venv/lib/python3.9/site-packages/urllib3/connectionpool.py"", line 445, in _make_request
    six.raise_from(e, None)
  File ""<string>"", line 3, in raise_from
  File ""/mnt/owid_live_covid/covid-19-data/scripts/venv/lib/python3.9/site-packages/urllib3/connectionpool.py"", line 440, in _make_request
    httplib_response = conn.getresponse()
  File ""/home/owid/.pyenv/versions/3.9.6/lib/python3.9/http/client.py"", line 1349, in getresponse
    response.begin()
  File ""/home/owid/.pyenv/versions/3.9.6/lib/python3.9/http/client.py"", line 316, in begin
    version, status, reason = self._read_status()
  File ""/home/owid/.pyenv/versions/3.9.6/lib/python3.9/http/client.py"", line 285, in _read_status
    raise RemoteDisconnected(""Remote end closed connection without""
http.client.RemoteDisconnected: Remote end closed connection without response

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File ""/mnt/owid_live_covid/covid-19-data/scripts/venv/lib/python3.9/site-packages/requests/adapters.py"", line 440, in send
    resp = conn.urlopen(
  File ""/mnt/owid_live_covid/covid-19-data/scripts/venv/lib/python3.9/site-packages/urllib3/connectionpool.py"", line 755, in urlopen
    retries = retries.increment(
  File ""/mnt/owid_live_covid/covid-19-data/scripts/venv/lib/python3.9/site-packages/urllib3/util/retry.py"", line 532, in increment
    raise six.reraise(type(error), error, _stacktrace)
  File ""/mnt/owid_live_covid/covid-19-data/scripts/venv/lib/python3.9/site-packages/urllib3/packages/six.py"", line 769, in reraise
    raise value.with_traceback(tb)
  File ""/mnt/owid_live_covid/covid-19-data/scripts/venv/lib/python3.9/site-packages/urllib3/connectionpool.py"", line 699, in urlopen
    httplib_response = self._make_request(
  File ""/mnt/owid_live_covid/covid-19-data/scripts/venv/lib/python3.9/site-packages/urllib3/connectionpool.py"", line 445, in _make_request
    six.raise_from(e, None)
  File ""<string>"", line 3, in raise_from
  File ""/mnt/owid_live_covid/covid-19-data/scripts/venv/lib/python3.9/site-packages/urllib3/connectionpool.py"", line 440, in _make_request
    httplib_response = conn.getresponse()
  File ""/home/owid/.pyenv/versions/3.9.6/lib/python3.9/http/client.py"", line 1349, in getresponse
    response.begin()
  File ""/home/owid/.pyenv/versions/3.9.6/lib/python3.9/http/client.py"", line 316, in begin
    version, status, reason = self._read_status()
  File ""/home/owid/.pyenv/versions/3.9.6/lib/python3.9/http/client.py"", line 285, in _read_status
    raise RemoteDisconnected(""Remote end closed connection without""
urllib3.exceptions.ProtocolError: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File ""/mnt/owid_live_covid/covid-19-data/scripts/src/cowidev/cmd/commons/get.py"", line 46, in run
    module.main()
  File ""/mnt/owid_live_covid/covid-19-data/scripts/src/cowidev/vax/incremental/russia.py"", line 80, in main
    Russia().export()
  File ""/mnt/owid_live_covid/covid-19-data/scripts/src/cowidev/vax/incremental/russia.py"", line 66, in export
    data = read(source).pipe(pipeline)
  File ""/mnt/owid_live_covid/covid-19-data/scripts/src/cowidev/vax/incremental/russia.py"", line 11, in read
    soup = get_soup(source)
  File ""/mnt/owid_live_covid/covid-19-data/scripts/src/cowidev/utils/web/scraping.py"", line 83, in get_soup
    response = get_response(source, request_method, use_proxy, **kwargs)
  File ""/mnt/owid_live_covid/covid-19-data/scripts/src/cowidev/utils/web/scraping.py"", line 49, in get_response
    raise err
  File ""/mnt/owid_live_covid/covid-19-data/scripts/src/cowidev/utils/web/scraping.py"", line 43, in get_response
    response = requests.get(source, **kwargs)
  File ""/mnt/owid_live_covid/covid-19-data/scripts/venv/lib/python3.9/site-packages/requests/api.py"", line 75, in get
    return request('get', url, params=params, **kwargs)
  File ""/mnt/owid_live_covid/covid-19-data/scripts/venv/lib/python3.9/site-packages/requests/api.py"", line 61, in request
    return session.request(method=method, url=url, **kwargs)
  File ""/mnt/owid_live_covid/covid-19-data/scripts/venv/lib/python3.9/site-packages/requests/sessions.py"", line 529, in request
    resp = self.send(prep, **send_kwargs)
  File ""/mnt/owid_live_covid/covid-19-data/scripts/venv/lib/python3.9/site-packages/requests/sessions.py"", line 645, in send
    r = adapter.send(request, **kwargs)
  File ""/mnt/owid_live_covid/covid-19-data/scripts/venv/lib/python3.9/site-packages/requests/adapters.py"", line 501, in send
    raise ConnectionError(err, request=request)
requests.exceptions.ConnectionError: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))
","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
cowidev.vax.incremental.saint_lucia,,,2022-10-11T07:03:18,,
cowidev.vax.incremental.serbia,5.19,True,2022-10-11T07:03:23,,
cowidev.vax.incremental.south_africa,7.03,True,2022-10-11T07:03:30,,
cowidev.vax.incremental.spain,4.44,True,2022-10-11T07:03:27,,
cowidev.vax.incremental.sri_lanka,,,2022-10-11T07:03:27,,
cowidev.vax.incremental.suriname,15.05,True,2022-10-11T07:03:42,,
cowidev.vax.incremental.taiwan,11.4,True,2022-10-11T07:03:41,,
cowidev.vax.incremental.thailand,3.48,True,2022-10-11T07:03:34,,
cowidev.vax.incremental.turkey,0.69,True,2022-10-11T07:03:34,,
cowidev.vax.incremental.united_arab_emirates,,,2022-10-11T07:03:34,,
cowidev.vax.incremental.vietnam,,,2022-10-11T07:03:34,,
cowidev.vax.incremental.who,4.88,True,2022-10-11T07:03:39,,
cowidev.vax.incremental.zambia,0.89,True,2022-10-11T07:03:40,,
